specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/VirtualNetworkGetWithServiceAssociationLink.json

```diff
@@ -0,0 +1,69 @@
+{
+	"parameters": {
+		"api-version": "2018-08-01",
+		"subscriptionId": "subId",
+		"resourceGroupName": "rg1",
+		"virtualNetworkName": "test-vnet"
+	},
+	"responses": {
+		"200": {
```

Comment:
nit: replace tab with 2 spaces and indent

Reply:
Could you clarify? Do you mean tab == 2 spaces (as opposed to 4)? These are already spaces, not tabs.

Reply:
Yep, 2 space tabs instead of 4